src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Collections/Item.java

```diff
@@ -0,0 +1,34 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Collections;
+
+public class Item {
+    private long id;
+    private String title;
+    private Type type;
+
+    public Item(long id, String title, Type type) {
+        this.id = id;
```

Comment:
please create default constructor where id will be generated automatically, after that reuse it into the overloaded constructor